pages/index.vue

```diff
@@ -147,15 +188,56 @@ export default defineComponent({
      * @todo 1. Send the request to update the task to the backend server.
      * @todo 2. Update the task in the dom.
      */
-    updateTask(_index, _id) {},
+    /*async */updateTask(_index, _id) {
+
+      let newTitle = this.todos[_index].title;
+      let token = this.$store.getters.token;
+      //console.log(newTitle);
+      let all = this;
+
+      if(newTitle!= '') {
+          /*await*/ this.$axios({
+            url: 'https://todo-app-csoc.herokuapp.com/' + "todo/" + _id + "/",
+            headers: {
+                Authorization: "Token " + token,
+            },
+            method: "patch",
+            data: {
+                id: _id,
+                title: newTitle,
+            }
+        }).then(function({data, status}) {
```

Comment:
if you would have used arrow function here then you wouldn't have required to store in `all`